extra/query_propagation.rst

```diff
@@ -0,0 +1,123 @@
+Controlling Query Propagation
+#############################
+
+When the user issues a query, the Citus coordinator partitions it into smaller query fragments where each query fragment can be run independently on a worker shard. This allows Citus to distribute each query across the cluster.
+
+However the way queries are partitioned into fragments (and which queries are propagated at all) varies by the type of query. In some advanced situations it is useful to manually control this behavior. Citus provides utility functions to propagate SQL to workers, shards, or placements.
```

Comment:
We can clarify here that this bypasses any coordinator logic, locking or other consistency checks. We should also note that these are in-place only as a last-resort to allow for statements which Citus otherwise does not run natively. If used incorrectly, it could lead to data inconsistency, deadlocks etc.